GDJS/Runtime/pixi-renderers/runtimegame-pixi-renderer.ts

```diff
@@ -72,6 +72,7 @@ namespace gdjs {
             (this._game.isAntialisingEnabledOnMobile() ||
               !gdjs.evtTools.common.isMobile()),
         });
+        this._threeRenderer.useLegacyLights = true;
```

Comment:
It's deprecated, but I guess it's ok to keep it for now. The change would mostly be to multiply lights intensity by Pi.